CHANGES/1026.doc.rst

```diff
@@ -0,0 +1,2 @@
+The pre-existing :meth:`/ <yarl.URL.__truediv__>` magic method
```

Comment:
Just realized that it's not obvious when it's only one char `/` that is clickable:
```suggestion
The pre-existing :meth:`/ magic method <yarl.URL.__truediv__>`
```